app/grandchallenge/evaluation/tasks.py

```diff
@@ -179,7 +179,11 @@ def prepare_and_execute_evaluation(
         evaluation.save()
         on_commit(evaluation.execute)
     else:
-        raise RuntimeError("No algorithm or predictions file found")
+        evaluation.update_status(
+            status=Evaluation.FAILURE,
+            error_message="An unexpected error has occurred.",
+        )
+        logger.error("No algorithm or predictions file found", exc_info=True)
```

Comment:
There is no exception information here to include.

```suggestion
        logger.error("No algorithm or predictions file found")
```